lib/foreman_maintain/concerns/downstream.rb

```diff
@@ -54,6 +54,10 @@ def package_name
         raise NotImplementedError
       end
 
+      def pkg_and_cmd_name
```

Comment:
You can keep pkg_and_cmd_name method in downstream concern only which should serve for both satellite and capsule features. With this I also think we should rename method to something like 'fm_pkg_and_cmd' ?